chrome/browser/net/chrome_network_delegate.cc

```diff
@@ -375,14 +375,15 @@ int ChromeNetworkDelegate::OnBeforeStartTransaction(
     content::ResourceType resource_type;
     if (info) {
       resource_type = info->GetResourceType();
-     bool isResourceTypeImage = (resource_type == content::RESOURCE_TYPE_IMAGE);
-      LOG(WARNING) << "Adblock: resource type of " << url << " is " << info->GetResourceType()
-                   << " (isImage = " << (isResourceTypeImage ? "true" : "false") << ")";
     } else {
-      LOG(WARNING) << "Adblock: No resourceRequestInfo";
-
+      LOG(WARNING) << "Adblock: No resourceRequestInfo, using regex detector";
+      resource_type = resource_type_detector_.detect(url);
     }
 
+    bool isResourceTypeImage = (resource_type == content::RESOURCE_TYPE_IMAGE);
+    LOG(WARNING) << "Adblock: resource type of " << url << " is " << resource_type
+                 << " (isImage = " << (isResourceTypeImage ? "true" : "false") << ")";
```

Comment:
Is this code (including the variable) only for debug? I'm not sure that it should stay, however, if so then could you please wrap it into a scope `{...}`?

Reply:
Yes, it's just for debugging purposes only. We have lot's of debug output and probably we should think about better solution.

The options are:
1) just remove it
2) generate .h file with different `bool` value for Debug and Release and introduce `ABPLOG` which actually uses `LOG` for Debug and silently returns for Release
3) ...

Reply:
Yeah, I'm fine with debugging messages, we can definitely elaborate `ABPLOG`, please let me know, I did it already at least a couple of times in other projects. However, since it includes a variable which is not used anywhere else, it would be better to put it in a scope.